saleor/core/utils/random_data.py

```diff
@@ -210,6 +210,9 @@ def create_products_by_schema(placeholder_dir, create_images):
     path = os.path.join(settings.PROJECT_ROOT, 'saleor', 'static', 'db.json')
     with open(path) as f:
         db_items = json.load(f, object_hook=object_hook)
+        for item in db_items:
+            if item["model"] in ["product.product", "product.productvariant"]:
```

Comment:
Could we move this logic to `create_products` and `create_product_variants` functions?

Reply:
Sure. good suggestion - gonna try and rebase